src/vector/feature.rs

```diff
@@ -72,12 +73,12 @@ impl<'a> Feature<'a> {
     /// If the field has an unsupported type, returns a [`GdalError::UnhandledFieldType`].
     ///
     /// If the field is null, returns `None`.
-    pub fn field(&self, name: &str) -> Result<Option<FieldValue>> {
-        let c_name = CString::new(name)?;
+    pub fn field<S: AsRef<str>>(&self, name: S) -> Result<Option<FieldValue>> {
```

Comment:
Same wrt. generic code.

Reply:
This is relatively small, but could still be split into a generic wrapper and a non-generic implementation.

Reply:
Sorry, I had actually fixed this, but it got lost between devices in a `git commit --amend`, sorry about that. The fix is back again.